src/openforms/forms/admin/tasks.py

```diff
@@ -0,0 +1,76 @@
+import logging
+import os
+import tempfile
+from pathlib import Path
+from uuid import uuid4
+from zipfile import ZipFile
+
+from django.conf import settings
+from django.core.files import File
+from django.core.mail import send_mail
+from django.core.management import call_command
+from django.template.loader import render_to_string
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from openforms.celery import app
+from openforms.utils.urls import build_absolute_uri
+
+from ..models import Form
+from ..models.form import FormsExport
+from ..utils import export_form
+from .tokens import exported_forms_token_generator
+
+logger = logging.getLogger(__name__)
+
+
+@app.task
+def process_forms_export(forms_uuids, email, username):
+    forms = Form.objects.filter(uuid__in=forms_uuids)
+
+    with tempfile.TemporaryDirectory(dir=settings.PRIVATE_MEDIA_ROOT) as temp_dir:
```

Comment:
does this clean up/delete the `temp_dir` after all the code here has been executed? If that is the case, please add a comment documenting this behaviour :)